osu.Game.Tests/Visual/Gameplay/TestSceneSkinEditor.cs

```diff
@@ -97,15 +100,10 @@ public void TestDragSelection()
                 InputManager.PressButton(MouseButton.Left);
             });
 
-            AddStep("Drag to bottom right", () =>
-            {
-                InputManager.MoveMouseTo(box3.ScreenSpaceDrawQuad.TopRight + new Vector2(-box3.ScreenSpaceDrawQuad.Width / 8, box3.ScreenSpaceDrawQuad.Height / 4));
-            });
+            AddStep("Drag to bottom right",
+                () => { InputManager.MoveMouseTo(box3.ScreenSpaceDrawQuad.TopRight + new Vector2(-box3.ScreenSpaceDrawQuad.Width / 8, box3.ScreenSpaceDrawQuad.Height / 4)); });
```

Comment:
Please refrain from making these types of reformats of otherwise unrelated code in PRs. They obscure the diff. I have to go check that this isn't a change in logic.

Reply:
Sorry, this happens whenever I refactor code with my IDE, I'll try to avoid this happening in the future.